posts/Yoann/2016-09-30-introducing-hibernate-search-sort-dsl.adoc

```diff
@@ -0,0 +1,88 @@
+= Introducing Hibernate Search Sort DSL
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search" ]
+:awestruct-layout: blog-post
+---
+With Elasticsearch support coming as a technological preview in Hibernate Search 5.6, you would think we're leaving out other features. Well, think again! Enters the Sort DSL, which will work with Elasticsearch of course, but also with the good ol' Lucene backend.
+
+The point here is to provide an API to build sort descriptions easily, without knowing everything about Hibernate Search features added on top of Lucene, such as `DistanceSortField`. And while we're at it, we're making it a modern, fluid API.
+
+== Overview
+
+The API works pretty much like the query builder. The query builder is the starting point, actually: a new `sort()` method has been added to the `QueryBuilder` interface. Here's what it looks like:
```

Comment:
I would just keep the sentence:
> A new `sort()` method has been added to the `QueryBuilder` interface:

Everything else seems redundant.

Additional point if you can avoid the passive form, something like the QueryBuilder interface now has an additional sort() method ...`